module1.js

```diff
@@ -87,13 +88,9 @@ function question1(name1) {
   @returns the product of num1 and num2 
 */
 
-// function question5(num1, num2) {
-//   // TODO
-
-//   return /* ??? */;
-// }
-
-
+function question5(num1, num2) {
+  return Math.round(num1 * num2, 2);
+}
 /**
```

Comment:
You could use...

return (( num1 * num2 ) * 100) / 100;

which removes the need for `Math.round()` and still corrects the JavaScript float error.